vertical-pod-autoscaler/enhancements/4016-in-place-updates-support/README.md

```diff
@@ -17,88 +17,135 @@
 
 ## Summary
 
-
 VPA applies its recommendations with a mutating webhook, during pod creation. It can also evict
-pods expecting that it will apply the recommendation when the pod is recreated. This is a
-disruptive process so VPA has some mechanism to avoid too frequent disruptive updates. 
+pods expecting that it will apply the recommendation when the pod is recreated. Today, this process
+is very disruptive as any changes in recommendations requires a pod to be recreated.
```

Comment:
Nit:

```suggestion
is potentially disruptive as any changes in recommendations requires a pod to be recreated.
```

The entire premise of running many pods along with a PDB is to reduce disruption, having a pod die should be something that workloads can handle.